### ISSUE TITLE
What is the inference speed?

### ISSUE THREAD

**Issue by qijiezhao** [ericsun99/Shufflenet-v2-Pytorch-#2]
Hi, I ran a script to test the inference speed:

```
net=shufflenetv2().cuda()
data=torch.autograd.Variable(torch.ones(8,3,224,224))
t1=time.time()
for i in range(100):
    _=net(data)
t2=time.time()
print((t2-t1)/float(100))
```
It gets ~4.4ms, slower than ~3ms, which is proposed in the original paper. 
So, how is the difference?

**Comment by Casablancaaa** [comment]
@qijiezhao  u can try to remove the warm up time .

**Comment by lxtGH** [comment]
I got the same problem, have you solved this problem

**Comment by HITerStudy** [comment]
> I got the same problem, have you solved this problem

Hello, why my test result was 20ms? Please give me some helps!Thanks.
